packages/phrases/src/locales/de/errors/resource.ts

```diff
@@ -0,0 +1,5 @@
+const resource = {
+  resource_identifier_in_use: 'Die API-Kennung {{indicator}} wird bereits verwendet',
```

Comment:
just double check we don't have the same phrase before? since you mentioned "follow the pattern" in the description

Reply:
No we don't. All the prev phrases were API exclusive.  e.g. The role name is already in use.  Could define a general error phrase once we define a general unique db field error type.  issue created under the eng quality project.

Reply:
okie